test/integration/serverless/src/logPython.py

```diff
@@ -1,4 +1,3 @@
-# flake8: noqa
```

Comment:
Just a heads up, removing these comments means that the Python test functions are going to need to comply with the agent's Python linter. It's not necessarily a bad thing, but you might have to make some more changes to these files to get it passing (or you could put the comments back).